chrome/content/zotero/preferences/preferences_export.xul

```diff
@@ -49,10 +50,15 @@
 			<label value="&zotero.preferences.quickCopy.defaultOutputFormat;" control="quickCopy-menu"/>
 			<menulist id="zotero-quickCopy-menu"/>
 			
-			<separator class="thin"/>
-			
-			<checkbox id="zotero-quickCopy-copyAsHTML" label="&zotero.preferences.quickCopy.copyAsHTML;"
-				oncommand="Zotero_Preferences.Export.buildQuickCopyFormatDropDown(document.getElementById('zotero-quickCopy-menu'), this.checked ? 'html' : '');"/>
+			<hbox align="center">
+				<label value="&zotero.preferences.quickCopy.locale;" control="locale-menu"/>
+				<menulist id="locale-menu"/>
```

Comment:
I'm not sure I see where you have generic code that's meant to work across multiple locale menus and that relies on the menulist id. If you do, that should accept the menulist object instead.